app/controllers/providers/confirm_dwp_non_passported_applications_controller.rb

```diff
@@ -5,14 +5,17 @@ class ConfirmDWPNonPassportedApplicationsController < ProviderBaseController
 
     def show
       delete_check_benefits_from_history
-      form
+      @form = Providers::ConfirmDWPNonPassportedApplicationsForm.new(model: partner)
     end
 
     def update
       return continue_or_draft if draft_selected?
 
-      if form.valid?
-        details_checked! if correct_dwp_result? && !details_checked?
+      @form = Providers::ConfirmDWPNonPassportedApplicationsForm.new(form_params)
```

Comment:
This sequence _may_ cause issues with save as draft

I'm not sure if save as draft works without the form being populated?

It might be worth checking?

Reply:
We are checking this with design, it looks like it does work without populating the form, but the way that it's handled seems very inconsistent across the service. However we did check 'save as draft' here and it works similarly to other pages, so I think it's something to look at separately from this ticket